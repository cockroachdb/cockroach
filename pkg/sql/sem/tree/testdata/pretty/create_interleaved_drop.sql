create table t (a int) interleave in parent p2 (i, "test") restrict
