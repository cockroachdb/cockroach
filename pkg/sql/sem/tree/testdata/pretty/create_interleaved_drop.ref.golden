// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
CREATE TABLE t (
	a
		INT8
)
INTERLEAVE IN PARENT p2 (
	i,
	test
) RESTRICT

10:
----------
CREATE TABLE t (
	a INT8
)
INTERLEAVE IN PARENT p2 (
	i,
	test
) RESTRICT

23:
-----------------------
CREATE TABLE t (a INT8)
INTERLEAVE IN PARENT p2 (
	i,
	test
) RESTRICT

42:
------------------------------------------
CREATE TABLE t (a INT8)
INTERLEAVE IN PARENT p2 (i, test) RESTRICT

66:
------------------------------------------------------------------
CREATE TABLE t (a INT8) INTERLEAVE IN PARENT p2 (i, test) RESTRICT


